src/toil/batchSystems/slurm.py

```diff
@@ -183,9 +183,13 @@ def prepareSbatch(self, cpu, mem, jobID):
         sbatch_line = ['sbatch', '-Q', '-J', 'toil_job_{}'.format(jobID)]
 
         if self.boss.environment:
+            comma = ''
+            ex = '--export='
             for k, v in self.boss.environment.iteritems():
                 quoted_value = quote(os.environ[k] if v is None else v)
-                sbatch_line.append('--export={}={}'.format(k, quoted_value))
+                ex = ex + ('{}{}={}'.format(comma, k, quoted_value))
+                comma = ','
```

Comment:
@evan-wehi you might want to consider using https://www.tutorialspoint.com/python/string_join.htm, it might simplify this logic.